iothub/service/src/Twin/Models/ClientTwin.cs

```diff
@@ -87,7 +87,7 @@ public ClientTwin(ClientTwinProperties twinProperties)
         /// Twin capabilities are read only.
         /// </remarks>
         [JsonPropertyName("capabilities")]
-        public ClientCapabilities Capabilities { get; internal set; }
+        public ClientCapabilities Capabilities { get; internal set; } = new();
```

Comment:
I got an NRE in a test in deserialization, so I gave more objects default instances.